<?xml version="1.0" ?>
<launch>

  <arg name="config_direct" default="" /> 

  <node pkg="mesapro" type="footpath_gesture_control.py" name="gesture_control" output="screen">
    <param name="config_direct" value="$(arg config_direct)"/>
  </node>

</launch>
